index.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <header >
+        <div class="container-main first-nav">
+                <ul class="white">
+                    
+                    <li><a href="#"><img src="./assets/mobile/facebook.png" alt="fb-logo" width="10px" height="10px"></a></li>
+                    <li><a href="#"><img src="./assets/mobile/twitter.png " alt="twit-log" width="10px" height="10px"></a></li>
+                    <li><a href="#">English</a></li>
+                    <li><a href="#">My Page</a></li>
+                    <li><a href="#">Log Out</a></li>
+                </ul>
+        </div>
+        <nav>
+            <div class="container-main">
+                <div class="container-start second-nav">
+                    <a href="#"><label for="logo"><img class="logo" src="./assets/mobile/logo1.png" alt="logo"</label></a>
+                    <ul>
+                        <li><a class="active" href="#">Home</a></li>
+                        <li><a class="active" href="./about.html">About</a></li>
+                        <li><a class="active" href="#courses">Courses</a></li>
+                        <li><a class="active" href="#teachers">Teachers</a></li>
+                        <li><a class="active" href="#partners">Partners</a></li>
+                        <li class="bb-btn">
+                            <button class="cc-btn">Al-Noor Campaign</button>
+                        </li>
+                    </ul>
+                </div>
```

Comment:
- [ ] This section should be presented more right as given in the design. 

Cloned
![image](https://user-images.githubusercontent.com/58504655/192089146-129600bd-654f-4ae2-a18a-efd4d73a467e.png)

Original 
![image](https://user-images.githubusercontent.com/58504655/192089154-4605f8b4-d938-414e-9e84-781230aa2c09.png)

Reply:
This one I agree i will change! Thanks